operator/main.go

```diff
@@ -197,6 +205,14 @@ func defineFlagVar() *FlagVar {
 	flag.BoolVar(&flagVar.enableModuleCatalog, "enable-module-catalog", true,
 		"Enabling the Module Catalog Synchronization for Introspection of "+
 			"available Modules based on ModuleTemplates.")
+	flag.BoolVar(&flagVar.enableKcpWatcher, "enable-kcp-watcher", true,
```

Comment:
By default should disable watcher
```suggestion
	flag.BoolVar(&flagVar.enableKcpWatcher, "enable-kcp-watcher", false,
```

Reply:
overwrite added [here](https://github.com/kyma-project/lifecycle-manager/pull/224/files#diff-28823c7f87387cdc0cd68c606bbca94a8e4c675690d3b9c6635a052a9b58caccR10-R16)